contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -317,14 +332,25 @@ contract LidoOracle is ILidoOracle, IsContract, AragonApp {
         // find mode value of this array
         (bool isUnimodal, uint256 mode) = Algorithm.mode(data);
         if (!isUnimodal)
-            return;
+            return (false, Report({beaconBalance: 0, beaconValidators: 0}));
 
         // unpack Report struct from uint256
         Report memory modeReport = uint256ToReport(mode);
 
+        return (true, modeReport);
+    }
+
+    /**
+     * @dev Pushed the current data point if quorum is reached
```

Comment:
A mistype here, you've probably meant Pushe**s**.

Reply:
Fixed